src/libbitcoind.cc

```diff
@@ -1566,7 +1566,10 @@ NAN_METHOD(GetMempoolOutputs) {
 
             Local<Object> output = NanNew<Object>();
 
-            output->Set(NanNew<String>("script"), NanNew<String>(script.ToString()));
+            output->Set(NanNew<String>("address"), NanNew<String>(psz));
```

Comment:
It's helpful to have the address in the object because when BWS and other services request outputs for multiple addresses, it can match which output goes with each address. I could put this code somewhere else, but it felt cleanest to put it here and have a consistent output format.